### DISCUSSION TITLE
Add GitHub Pages As One Of The Categories At This Forum

### DISCUSSION THREAD

**Discussion by jfullstackdev** [community/community#21328] [Other Feature Feedback, Questions, & Ideas] [answered]
In the old `github.community`, GitHub Pages is one of the most asked questions, please add this category at this Forum. Thanks.

@tuves

**Comment by tuves** [comment, marked as answer]
Hi @jdevstatic 👋🏽 ,

Happy to see you here! Some of our Discourse categories have changed during the move. Check out [this post](https://github.com/orgs/community/discussions/21279) to see the new and improved names. GitHub Pages is just under a different category, called New to GitHub :)

**Reply by jfullstackdev** [reply]
I think that will be great if it is on the top level

and be added at the Labels too

thanks

**Reply by tuves** [reply]
Thanks for the feedback, @jdevstatic. Will note it and share it with the team.

**Reply by jfullstackdev** [reply]
update : I already saw `Pages` label, I was typing GitHub Pages

**Reply by jfullstackdev** [reply]
finally other than the Pages label, it's now at the Categories too !

that's a great addition, as I'm now seeing so much post about GitHub Pages like in the old Community Forum, there are so many needing this help

it's one of the main feature of GitHub, so yes, this is a great addition for more visibility

thanks for that 

![image](https://user-images.githubusercontent.com/47092464/186655125-531ebced-0182-4b41-8d5b-67d8aac1a97d.png)
